app/models.py

```diff
@@ -340,3 +340,50 @@ class UserSkill(db.Model): #pylint: disable=no-init,too-few-public-methods
     user_id = Column(types.Integer, ForeignKey('users.id'), nullable=False)
 
     constraint = UniqueConstraint('user_id', 'name')
+
+class Event(db.Model):
+    '''
+    An event that shows up in the activity feed for a deployment.
+    '''
+
+    __tablename__ = 'events'
+    id = Column(types.Integer, autoincrement=True, primary_key=True)  #pylint: disable=invalid-name
+    created_at = Column(types.DateTime(), default=datetime.datetime.now)
+    updated_at = Column(types.DateTime(), default=datetime.datetime.now,
+                        onupdate=datetime.datetime.now)
+    type = Column(types.String)
+
+    user_id = Column(types.Integer, ForeignKey('users.id'))
```

Comment:
er, also, I didn't specify `nullable` on the `user_id` field since I assumed `user_id` could be null, but that proves problematic too since there would be no way to figure out what deployment an event is for. So perhaps `deployment` *is* needed as an explicit field...